extensions/sql-migration/src/wizard/databaseBackupPage.ts

```diff
@@ -1299,24 +1298,74 @@ export class DatabaseBackupPage extends MigrationWizardPage {
 				});
 				this.migrationStateModel._sourceDatabaseNames = this.migrationStateModel._databasesForMigration;
 
+				const createTableHeaderComponents = (headerNames: { name: string; requiredIndicator: boolean }[]) =>
+					headerNames.map(header =>
+						this._view.modelBuilder.divContainer().withItems([
+							this._view.modelBuilder.divContainer().withProps({
+								CSSStyles: { marginTop: '-1em' }
+							}).component(),
+							this._view.modelBuilder.text().withProps({
+								value: header.name,
+								requiredIndicator: header.requiredIndicator,
+								CSSStyles: {
+									...styles.LABEL_CSS,
+									margin: '0'
+								}
+							}).component(),
+							this._view.modelBuilder.divContainer().withProps({
+								CSSStyles: { marginBottom: '-1em' }
+							}).component()
+						]).component()
+					);
+
+				let networkShareTargetTableHeaderNames = [
+					{ name: constants.SOURCE_DATABASE, requiredIndicator: false },
+					{ name: constants.TARGET_DATABASE_NAME, requiredIndicator: true },
+					{ name: constants.NETWORK_SHARE_PATH, requiredIndicator: true },
+				]
+
+				let networkShareTargetTableHeaderComponents = createTableHeaderComponents(
+					networkShareTargetTableHeaderNames
+				);
+
 				const networkShareTargetData = this.migrationStateModel._databasesForMigration
 					.map((db, index) => [
-						{ value: db },
+						{ value: this._view.modelBuilder.text().withProps({ value: db, CSSStyles: { 'margin': '0' } }).component() },
 						{ value: this._networkShareTargetDatabaseNames[index] },
 						{ value: this._networkShareLocations[index] }]);
-				await this._networkShareTargetDatabaseNamesTable.setDataValues(networkShareTargetData);
+				await this._networkShareTargetDatabaseNamesTable.setDataValues([
+					networkShareTargetTableHeaderComponents.map(component => ({ value: component })),
```

Comment:
This will lead to another accessibility issue since now table headers won't be announced as table headers.

Reply:
Did this way as we do not have an option to give indicators in headers. Adding * was not a very good way and it would be shown in black which will be different from other indicators in extension leading to inconsistency in UI. Do you have a workaround for this?

Reply:
As discussed, merging this as a temporary fix.